### DISCUSSION TITLE
PRIMUS gnom P(r) - changing Guinier Rg / I(0) values

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1944] [Primary data processing] [answered]
Hello,

I am using ATSAS 3.2.1 to analyse some SAXS data, in particular the primus 'Distance Distribution Wizard' for determination of the p(r) distribution and the Dmax. In the GUI, a value for the Guinier Rg /I(0) is shown as well as a value for the p(r) Rg/I(0). When I vary the Rmax value I see the p(r) Rg/I(0) change, which I expect. However I also see the Guinier Rg/I(0) vary with different Rmax values (when the range of datapoints being included is the same). Why does this happen? I thought the Guinier Rg should be fixed (and the same as what I determined using the 'Radius of Gyration' wizard)?

Thank you.

**Comment by franke-biosaxs** [comment, marked as answer]
> However I also see the Guinier Rg/I(0) vary with different Rmax values (when the range of datapoints being included is the same). Why does this happen? I thought the Guinier Rg should be fixed (and the same as what I determined using the 'Radius of Gyration' wizard)

The Guinier Rg is calculated from the reconstructed data. As the data is reconstructed based on the p(r), if the p(r) changes, the calculated scattering changes and with it, in general, the Rg and I(0) values.
